docs/en/ingest-management/fleet/fleet-server-scaling.asciidoc

```diff
@@ -0,0 +1,300 @@
+[discrete]
+[[fleet-server-scalability]]
+== {fleet-server} scalability
+
+This section summarizes the resource and {fleet-server} configuration
+requirements needed to scale your deployment of {agent}s. To scale
+{fleet-server}, you need to modify settings in your deployment and the
+{fleet-server} agent policy.
+
+First modify your {fleet} deployment settings in {ecloud}:
+
+. Log in to {ecloud} and go to your deployment.
+
+. Under *Deployments > _deployment name_*, click *Edit*.
+
+. Under APM & Fleet:
++
+--
+* Modify the compute resources available to the server to accommodate a higher
+scale of {agent}s
+* Modify the availability zones to satisfy fault tolerance requirements
+
+For recommended settings, refer to <<scaling-recommendations>>.
+
+[role="screenshot"]
+image::images/fleet-server-hosted-container.png[{fleet-server} hosted agent]
+--
+
+Next modify the {fleet-server} configuration by editing the agent policy: 
+
+. In *Fleet*, click *Agent Policies*. Click on the *{ecloud} agent policy* to
+edit it.
+
+. Open the *Actions* menu and select *Edit integration*.
++
+[role="screenshot"]
+image::images/elastic-cloud-agent-policy.png[{ecloud} policy]
+
+. Under {fleet-server}, modify *Max Connections* and other
+<<fleet-server-configuration,advanced settings>> as described in
+<<scaling-recommendations>>. 
++
+[role="screenshot"]
+image::images/fleet-server-configuration.png[{fleet-server} configuration]
+
+[discrete]
+[[fleet-server-configuration]]
+=== Advanced {fleet-server} options
+
+The following advanced settings are available to fine tune your {fleet-server}
+deployment.
+
+//TODO: We need to discuss the best way to format config settings. I'm using
+//mostly tables in other sections of the docs for improved scanning, but that is
+//controversial.
+
+//Nima: I tried removed repetitive words and tried clarify the descriptions here,
```

Comment:
ok if you publish. then I can check against the source and correct
Would that work?